### ISSUE TITLE
Estimating PRS uncertainty

### ISSUE THREAD

**Issue by mglev1n** [getian107/PRScsx#25]
LDpred2 allows the user to return samples from the posterior, which can then be combined with genotypes in order to estimate individual-level uncertainty in polygenic risk (https://privefl.github.io/bigsnpr/articles/prs_uncertainty.html and https://rdcu.be/c6bmI). Since PRScs/PRScsx also use MCMC, is it possible to enable similar functionality? It seems like this section is probably relevant: https://github.com/getian107/PRScsx/blob/5f27ff89576b9b052bb4a25ecd165e383ffb113d/mcmc_gtb.py#L111

**Comment by getian107** [comment]
Yes- The output file will be large if we write out all posterior samples but technically it's straightforward. I can incorporate this functionality in the next release.

**Comment by getian107** [comment]
Just a quick note that PRScs now has the option to output all posterior samples, which can be used to estimate PRS uncertainty.
